server/lib/routes/subscription.js

```diff
@@ -3,18 +3,9 @@
 const S = require('fluent-json-schema')
 
 const subscriptionMap = new Map()
-subscriptionMap.set('web', 'endpoint')
 subscriptionMap.set('expo', 'token')
 
 const bodySchema = S.anyOf([
```

Comment:
We can get rid of  `S.anyOf` here since now we only expect a single type(expo) of payload